app/views/DocumentingView.vue

```diff
@@ -328,9 +326,9 @@
         subTitle="Custom component example"
       >
         <p>
-          By default, Vue-Doxen displays a text area on the demo page for users to type
-          whatever they want into it, and it will be passed along to the slot. However,
-          you can replace the textarea with your own custom component.
+          By default, Vue-Doxen displays a <code>textarea</code> on the demo page for
+          users to type whatever they want into it, and it will be passed along to the
+          slot. However, you can replace the textarea with your own custom component.
```

Comment:
```suggestion
          By default, Vue-Doxen displays a <code>textarea</code> on the demo page for
          users to type whatever they, and it will be passed along to the           slot. However,
          you can replace the <code>textarea</code> with your own custom component.
```